src/style.css

```diff
@@ -0,0 +1,24 @@
+body {
+  margin: 20px 10px;
+  background-color: black;
+}
+
+.block {
+  margin: 10px;
+  border: 1px solid #e95d49;
+}
+
+.first {
+  margin-bottom: 40px;
+  width: 348px;
```

Comment:
the width should be 350px

Reply:
use box-sizing rule in this task
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing